genesyscloud/telephony_providers_edges_site/resource_genesyscloud_telephony_providers_edges_site_schema.go

```diff
@@ -272,6 +272,13 @@ func ResourceSite() *schema.Resource {
 				Default:     false,
 				Type:        schema.TypeBool,
 			},
+			"managed": {
```

Comment:
Hey @bbbco going to approve the ticket.  Just wondering why we are explictly adding a managed attribute to the resource. The user will never be able to set it and if we query the resource we can see that is managed.  Why add a managed resource attribute when we can check to see if the resource is managed.  Is this just being put here to eliminate API calls.

Reply:
Yeah, it's primarily for export purposes. We need to determine if the resource is managed or not, but this is something that is never managed by the user (only by the API). We can refer to it in the `ExportAsDataFunc` without making a separate API call if we just store it in the state when it is first read.